app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,66 @@
+require "gds_api/account_api"
+
+class SessionsController < ApplicationController
+  include AccountConcern
+
+  before_action :set_no_cache_headers
+
+  def create
+    redirect_with_ga account_manager_url and return if logged_in?
+
+    redirect_with_ga GdsApi.account_api.get_sign_in_url(
+      redirect_path: params[:redirect_path],
+      state_id: params[:state],
```

Comment:
[Use `state_id` for the param name too](https://github.com/alphagov/govuk-rfcs/blob/main/rfc-134-govuk-wide-session-cookie-and-login.md#extend-frontend-to-manage-the-auth-process).

Reply:
Done here: https://github.com/alphagov/frontend/pull/2668/commits/ebb7aa9d44df4229db2aa27904d65f999166731d
Thanks!